proposed/simplecache.md

```diff
@@ -0,0 +1,207 @@
+characters and encodings or longer lengths, but must support at least that
+minimum.  Libraries are responsible for their own escaping of key strings
+as appropriate, but MUST be able to return the original unmodified key string.
+The following characters are reserved for future extensions and MUST NOT be
+supported by implementing libraries: `{}()/\@:`
+
+*    **Cache** - An object that implements the `Psr\SimpleCache\CacheInterface` interface.
+
+*    **Cache Misses** - A cache miss will return null and therefore detecting
+if one stored null is not possible. This is the main deviation from PSR-6's
+assumptions.
+
+
+### 1.3 Cache
+
+Implementations MAY provide a mechanism for a user to specify a default TTL
+if one is not specified for a specific cache item.  If no user-specified default
+is provided implementations MUST default to the maximum legal value allowed by
+the underlying implementation.  If the underlying implementation does not
+support TTL, the user-specified TTL MUST be silently ignored.
+
+
+2. Interfaces
+-------------
+
+### 2.1 CacheInterface
+
+The cache interface provides the most basic functionality of cache servers which
+entails basic reading, writing and deleting of single cache items.
+
+In addition it has methods for dealing with multiple sets of cache entries such as writing, reading or
+deleting multiple cache entries at a time. This is useful when you have lots of cache reads/writes
+to perform, and lets you perform your operations in a single call to the cache server cutting down latency
+times dramatically.
+
+Finally for counters it provides the ability to increment and decrement a cache key
+atomically.
+
+```php
+<?php
+
+namespace Psr\SimpleCache;
+
+interface CacheInterface
+{
+    const TTL_MINUTE = 60;
+    const TTL_HOUR = 3600;
+    const TTL_DAY = 86400;
+
+    /**
+     * Fetched a value from the cache.
+     *
+     * @param string $key The unique key of this item in the cache
+     *
+     * @return mixed The value of the item from the cache, or null in case of cache miss
+     */
+    public function get($key);
+
+    /**
+     * Persist data in the cache, uniquely referenced by a key with an optional expiration TTL time.
+     *
+     * @param string $key The key of the item to store
+     * @param mixed $value The value of the item to store
+     * @param null|integer|DateInterval $ttl Optional. The TTL value of this item. If no value is sent and the driver supports TTL
+     *                                       then the library may set a default value for it or let the driver take care of that.
+     *
+     * @return bool True on success and false on failure
+     */
+    public function set($key, $value, $ttl = null);
+
+    /**
+     * Delete an item from the cache by its unique key
+     *
+     * @param string $key The unique cache key of the item to delete
+     *
+     * @return bool True on success and false on failure
```

Comment:
Could probably also be a no-return operation, since many underlying layers may not even report a success or failure. Wondering what the consumer-side use-case for knowing whether a delete succeeded or not could be...